setup.py

```diff
@@ -11,14 +11,14 @@
 NAME = 'pactum'
 VERSION = '0.0.1'
 DESCRIPTION = 'Create API specifications and documentation using Python'
-URL = 'https://github.com/osantana/pactum'
-EMAIL = 'pactum@osantana.me'
+URL = 'https://github.com/olist/pactum'
+EMAIL = 'pactum@olist.com'
 AUTHOR = 'Osvaldo Santana Neto'
```

Comment:
just Osvaldo?

Reply:
correios lib is like this (just him) some others have `Olist Developers` as author.